# RWA4
#drop parts: no
#new order: no
#flipped part: yes
#faulty parts: yes
#faulty gripper: no
#moving obstacles: yes (2: aisles 2 and 3)
#sensor blackout: yes

options:
  insert_models_over_bins: false
  insert_models_over_shelves: true
  belt_population_cycles: 0

shelf_layout:
  row_1: ['base','collar',0,'base'] #shelves: 3, 4, 5
  row_2: ['pipe',0,'base','base']   #shelves: 6, 7, 8
  row_3: ['base','pipe',0,'collar'] #shelves: 9, 10, 11

aisle_layout:
  person_1:
    location: -1.57
    start_time: 0.
    move_time: 9.
    wait_time: 7.
  person_2:
    location: 1.57
    start_time: 16.
    move_time: 9.
    wait_time: 7.

orders:
  order_0:
    announcement_condition: time
    announcement_condition_value: 0.0
    shipment_count: 1
    destinations: [agv2]
    products:
      product_0:
        type: pulley_part_blue
        pose:
          xyz: [0.0, 0.1, 0]
          rpy: ['pi', 0, 0]
      product_1:
        type: piston_rod_part_red
        pose:
          xyz: [-0.1, -0.2, 0]
          rpy: [0, 0, '-pi/4']


models_over_bins:
  bin13:
    models:
      pulley_part_blue:
        xyz_start: [0.1, 0.1, 0.0]
        xyz_end: [0.45, 0.45, 0.0]
        rpy: [0, 0, 0]
        num_models_x: 2
        num_models_y: 1
  bin1:
    models:
      piston_rod_part_red:
        xyz_start: [0.1, 0.1, 0.0]
        xyz_end: [0.5, 0.5, 0.0]
        rpy: [0, 0, pi/4]
        num_models_x: 2
        num_models_y: 2

models_over_shelves:
  shelf8:
    models:
      gasket_part_green:
        xyz_start: [0.5, 0.4, 0.0]
        xyz_end: [1.4, 0.4, 0.0]
        rpy: [0, 0, -pi/4]
        num_models_x: 6
        num_models_y: 1

sensor_blackout:
  product_count: 1
  duration: 25

faulty_products:
  - pulley_part_blue_1
  # - pulley_part_blue_2
  - piston_rod_part_red_2
  - piston_rod_part_red_3
  - gasket_part_green_2
  - gasket_part_green_5
time_limit: -1
